jep/207/README.adoc

```diff
@@ -85,15 +85,17 @@ Support of other logging types will be documented in subsequent JEPs.
 * Minimize log traffic between Jenkins agents and the master, logs should be reported from slaves directly when it is possible
 * External Build Log Storage has an extensible and pluggable architecture
 * Reference implementations:
-   * File-system-based storage (current implementation)
-   * One external build log storage based on the open-source stack, based on Fluentd/Logstash and CloudWatch for AWS
-   * Nice2Have: Elasticsearch-Logstash-Kibana for self-hosted solutions
-* Jenkins Pipeline is supported out of the box
-* Real-time and Annotated run console logs should be still supported
-* Data migration flow for upgrading instances
+** File-system-based storage (current implementation)
+** One external build log storage based on the open-source stack, e.g. Elasticsearch-Logstash-Kibana (ELK)
+** Nice 2 have: one Cloud-focused implementation, e.g. for AWS CloudWatch
+* Freestyle and Jenkins Pipeline projects are supported out of the box
+* Run console logs should be provided using standard Jenkins interfaces
+* Data migration flow for upgrading instances is technically possible
+** It is OK if it requires special actions outside Jenkins
 * Reference External Build logging implementation provides a feasible performance and fault-tolerance compared to the original filesystem based solution
-* All Global Configurations are designed in a way that they can be configured via Jenkins Configuration-as-Code Plugin OOTB[e][f]
-Presumptions, obstacles and limitations (braindump)
+* All Global Configurations are designed in a way that they can be configured via Jenkins Configuration-as-Code Plugin
+
+=== Design decisions
```

Comment:
Better to follow the standard JEP structure: **Reasoning** and **Motivation**.

Reply:
I really cannot read JEPs when decision making goes after the specification. I would prefer to keep it here unless it violates JEP requirements

Reply:
@oleg-nenashev 
The choice of top level sections and their order was discussed and agreed on.  It is set and should not be changed.     If it matters to you that it be another way, propose the change i
including bringing all other JEPs into compliance with the new structure.  


It was an mistake on my part as an editor that I didn't catch that this JEP does not meet the requirements.  I'll make the updates to bring it back into compliance.

Reply:
@oleg-nenashev  
Looking now, I see that the reasoning for the section ordering is not mentioned in the JEP-1, but I know it was discussed.  I will go dig up the discussion and make sure it is added to the Reasoning.

Reply:
Anyway, it is quite unrelated to this PR, it does not really change it.
So I would prefer to merge it as is and then discuss.